app/projectsmodel.h

```diff
@@ -88,7 +88,24 @@ class ProjectModel : public QAbstractListModel
 
       bool operator < ( const ProjectFile &str ) const
       {
-        return ( folderName < str.folderName );
+        if ( !projectNamespace.isEmpty() && str.projectNamespace.isEmpty() )
+        {
+          return false;
+        }
+        if ( projectNamespace.isEmpty() && !str.projectNamespace.isEmpty() )
+        {
+          return true;
+        }
+
+        if ( ( projectNamespace.isEmpty() && str.projectNamespace.isEmpty() ) || projectNamespace.toLower() == str.projectNamespace.toLower() )
```

Comment:
is it necessary to have also the check for isEmpty() here? doesn't the second check cover that too?